src/docs/_sidebars/arcCommanderManualSidebar.md

```diff
@@ -1,17 +1,32 @@
 ---
 ---
 
+```ARC Commander Setup 
+# Installing Dependencies:/docs/implementation/ArcCommanderManual/arc_installation_dependencies.html
+# ARC User configuration:/docs/implementation/ArcCommanderManual/arc_config.html
+# Installing the ARC Commander:/docs/implementation/ArcCommanderManual/arc_installation.html
+## Windows Installation:/docs/implementation/ArcCommanderManual/arc_installation_windows.html
+## MacOS Installation:/docs/implementation/ArcCommanderManual/arc_installation_macos.html
+## Linux Installation:/docs/implementation/ArcCommanderManual/arc_installation_linux.html
+# Before we start:/docs/implementation/ArcCommanderManual/arc_usage_notes.html
+```
+
 ```ARC Commander Manual
```

Comment:
The headlines can be improved as Arc Commander Manual is duplicated. Suggetsion: 
ARC Commander 
DataHUB Setup

and for ISA Metadata:
ARC Commander
ARC setup or ARC metadata setup
ISA Metadata might be to specific for first time users and can follow as a subheading.

Reply:
I don‘t think, that „manual“ is duplicated, but this might be hard to see on the PR. 

And good point - also not sure yet for me, whether this is supposed to target first timers.

Reply:
You reach the manual via "Arc Commander" and klick "Manual" in the line below in the main sidebar.
Than you have the whole manuaal split in three major sections. The middle one again, has the headling "Arc Commander" and within this level "Manual" but also in the line below.
WHile we are in the Arc Commander Manual we have no a second Arc Commander Manual opening up.. This is what I meant with duplicated.

Reply:
Ah now I got it. Good point. I did not find a good word for the middle section and yes the setup and isa sections are also part of the manual.

Reply:
Any name suggestions for the two sections? 
Does it make sense to separate the ISA section anyways? My thought was to have functions "around" the ARC separated from those specifically writing on ISA files ("within" the ARC).

Reply:
Adapted the sections and added a pre-section "Manual"